### PR TITLE
Invenio v1.0.10

### PR DESCRIPTION
**Pull request by tiborsimko**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -6,6 +6,58 @@ releases.  For more information about the current release, please
...
+    the old `tiborsimko/invenio-devscripts` repository of helper
+    scripts and adapted to resemble Invenio 3 kickstart installation
+    scripts. Tested on Ubuntu 12.04 and CentOS 6.
+
+Improved features
+~~~~~~~~~~~~~~~~~
+
++ I18N
+
+  - Updates Catalan, French, German, Italian, Russian, Slovak Spanish
```
Comment (egabancho) [inline NEWS]: I don't remember if the release notes are auto generated or not for `main-1.x` but, I guess there is an "and" missing between Slovak and Spanish

File: RELEASE-NOTES [hunk 2]
```diff
@@ -11,52 +11,63 @@ Invenio is a digital library framework enabling you to build your own
...
+    scripts and adapted to resemble Invenio 3 kickstart installation
+    scripts. Tested on Ubuntu 12.04 and CentOS 6.
 
 Improved features
 -----------------
 
-+ installation:
++ I18N
+
+  - Updates Catalan, French, German, Italian, Russian, Slovak Spanish
```
Comment (egabancho) [inline RELEASE-NOTES]: sames as https://github.com/inveniosoftware/invenio/pull/3713/files#r87218469

File: NEWS (hunk 1 above)
Comment (tiborsimko) [inline NEWS]: Yes, they are auto-generated... Amended by amended commit log message, thanks :smile:

**Comment by egabancho** [comment]
LGTM

**Pull request merged**